tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
     /* Language and Environment */
-    "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    "target": "es2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
Makes sense to me.
If you've made a change after reading something in future, feel free to put the link to what you read in the commit message!